03_Your_first_AI_training_job_on_LUMI/reference_solution/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#SBATCH --account=project_465001063
+#SBATCH --partition=small-g
+#SBATCH --gpus-per-node=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=7
+#SBATCH --mem-per-gpu=60G
+#SBATCH --time=0:15:00
+
+# Set up the software environment
+# NOTE: these modules will be available from the LUMI system stack after July 2024 and the "module use" line will no longer work
+module purge
```

Comment:
Note the remark about using `module purge` with the sticky software stack modules on LUMI in the [LUMI Docs](https://docs.lumi-supercomputer.eu/runjobs/lumi_env/Lmod_modules/#loading-and-unloading-modules). I don't know if `module --force purge` is a good or bad idea here.